crypto/keys/types.go

```diff
@@ -196,6 +205,54 @@ func (i offlineInfo) GetPath() (*hd.BIP44Params, error) {
 	return nil, fmt.Errorf("BIP44 Paths are not available for this type")
 }
 
+type multisigPubKey struct {
+	PubKey crypto.PubKey `json:"pubkey"`
+	Weight uint          `json:"weight"`
+}
+type multiInfo struct {
+	Name      string           `json:"name"`
+	PubKey    crypto.PubKey    `json:"pubkey"`
+	Threshold uint             `json:"threshold"`
+	PubKeys   []multisigPubKey `json:"pubkeys"`
+}
+
+func NewMultiInfo(name string, pub crypto.PubKey) Info {
```

Comment:
I think this needs `Stringer` too.

Reply:
It does? Why? None of the other `Info` types have `Stringer` implemented. They all use the `Bech32*` output functions.

Reply:
🤦‍♂️ thank you for catching that totally right. Saw this and forgot to go back and delete this comment.